src/clients/oauth.rs

```diff
@@ -253,7 +253,7 @@ pub trait OAuthClient: BaseClient {
     async fn playlist_replace_tracks<'a>(
         &self,
         playlist_id: &PlaylistId,
-        track_ids: impl IntoIterator<Item = &'a TrackId> + Send + 'a,
+        track_ids: impl IntoIterator<Item = &'a dyn PlayableId> + Send + 'a,
```

Comment:
Since it's an endpoint to manipulate track, it doesn't need to be  `dyn PlayableId`. `TrackId` is enough, right?

Reply:
I've just made a commit that leaves this and similar occurrences as a TODO for another PR: https://github.com/ramsayleung/rspotify/pull/244/commits/98c299c14505c08ebe14e84061a1155f45511c3d. This is because most of the playlist endpoints work with both tracks *and* episodes, and we don't currently take that into account. I left it as a TODO to avoid making big changes so that this can be merged finally.